StitchCore/StitchCore/Source/Auth/AuthInfo.swift

```diff
@@ -14,13 +14,13 @@ internal struct AuthInfo: Codable {
     let accessToken: DecodedJWT?
 
     // The user this session was created for.
-    let deviceId: String
+    let deviceId: String?
```

Comment:
[q] What happened here?

Reply:
An offline conversation will be had to reject DRY principles here in favor of creating a separate struct for linking.